docs/resources/instance_pool.md

```diff
@@ -45,8 +45,7 @@ The following arguments are required:
 * `node_type_id` - (Required) (String) The node type for the instances in the pool. All clusters attached to the pool inherit this node type and the pool’s idle instances are allocated based on this type. You can retrieve a list of available node types by using the [List Node Types API](https://docs.databricks.com/dev-tools/api/latest/clusters.html#clusterclusterservicelistnodetypes) call.
 * `custom_tags` - (Optional) (Map) Additional tags for instance pool resources. Databricks tags all pool resources (e.g. AWS & Azure instances and Disk volumes). *Databricks allows at most 43 custom tags.*
 * `enable_elastic_disk` - (Optional) (Bool) Autoscaling Local Storage: when enabled, the instances in the pool dynamically acquire additional disk space when they are running low on disk space.
-
-* `preloaded_spark_versions` - (Optional) (List) A list with the runtime version the pool installs on each instance. Pool clusters that use a preloaded runtime version start faster as they do have to wait for the image to download.  You can retrieve them via [databricks_spark_version](../data-sources/spark-version.md) data source or via  [Runtime Versions API](https://docs.databricks.com/dev-tools/api/latest/clusters.html#clusterclusterservicelistsparkversions) call.
+* `preloaded_spark_versions` - (Optional) (List) A list with at most one runtime version the pool installs on each instance. Pool clusters that use a preloaded runtime version start faster as they do have to wait for the image to download. You can retrieve them via [databricks_spark_version](../data-sources/spark-version.md) data source or via  [Runtime Versions API](https://docs.databricks.com/dev-tools/api/latest/clusters.html#clusterclusterservicelistsparkversions) call.
```

Comment:
I removed the newline above so the changes isn't as clear, but in essence I changed the following.

```diff
- A list with the runtime version the pool installs on each instance.
+ A list with at most one runtime version the pool installs on each instance.
```

Reply:
`as they do have` - > `as they do not have`

Reply:
Change made. Note that the databricks docs also have this issue, but I don't think there's a place I can contribute a fix to.

<img width="875" alt="Screen Shot 2021-05-03 at 1 49 48 PM" src="https://user-images.githubusercontent.com/4990214/116912607-73621d80-ac16-11eb-88af-06be2c5ffe7a.png">